src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -47,6 +48,16 @@ const formConfig = {
   trackingPrefix: '10-10D-',
   introduction: IntroductionPage,
   confirmation: ConfirmationPage,
+  v3SegmentedProgressBar: true,
+  preSubmitInfo: {
+    statementOfTruth: {
+      body:
+        'I confirm that the identifying information in this form is accurate and has been represented correctly.',
+      messageAriaDescribedby:
+        'I confirm that the identifying information in this form is accurate and has been represented correctly.',
+      fullNamePath: formData => getNameKeyForSignature(formData),
+    },
+  },
```

Comment:
Adds the statement of truth box + signature field to the end of the form and verifies that the user types one of three possible name values based on the result of `getNameKeyForSignature`